Refactor action creators (#144)
lundegaard/redux-tools: docs/packages/actions.md

This package is a collection of utility functions for creating [FSA-compliant](https://github.com/redux-utilities/flux-standard-action) action creators and reducers.

## Best Practices

> Only use `makeEmptyActionCreator`, `makePayloadActionCreator`, and `makePayloadMetaActionCreator` in your applications. Do not use any other action creator factories in your own code. Use the appropriate action creator factory based on the number of arguments the action creator should expect (i.e. `Constant` for zero, `Simple` for one, and `PayloadMeta` if any metadata can be passed).

Reviewer: ```suggestion
Only use `makeEmptyActionCreator`, `makePayloadActionCreator`, and `makePayloadMetaActionCreator` in your applications. Do not use any other action creator factories in your own code. Use the appropriate action creator factory based on the number of arguments the action creator should expect (i.e. `Empty` for zero, `Payload` for one, and `PayloadMeta` if any metadata can be passed).
```

( _Object_ ): Object with values set to `<prefix>/<type>`.

### makeEmptyActionCreator()

Creates a new nullary action creator.

**Arguments**

1. `type` ( _string_ ): The action type.

**Returns**

( _() -> Action_ ): An action creator.

### makePayloadActionCreator()

Creates a new unary action creator which will use the argument as the payload.

**Arguments**

1. `type` ( _string_ ): The action type.

**Returns**

( _any -> Action_ ): An action creator.

### makePayloadMetaActionCreator()

Creates a new binary action creator which will use the first argument as the payload and the second argument as the meta.

**Arguments**

1. `type` ( _string_ ): The action type.

**Returns**

( _(any, Object) -> Action_ ): An action creator.

### configureActionCreator()

!> This action creator factory is only meant to be used to create other action creator factories. We do not recommend using makeActionCreator in your application code.

> !>

Reviewer: is having two !> necessary?

Creates a new unary action creator which will apply the provided functions to an argument, producing
the `payload` and `meta` properties.

**Arguments**

1. `type` ( _string_ ): The action type.

> 2. `getPayload` ( _any –> any_ ): Payload getter.

Reviewer: ```suggestion
2. `getPayload` ( _(...any) –> any_ ): Payload getter.
```

> 3. `getMeta` ( _any -> any_ ): Meta getter.

Reviewer: ```suggestion
3. `getMeta` ( _(...any) -> any_ ): Meta getter.
```
